lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1654,6 +1654,18 @@ void AtenMaskedFillTensorOp::getCanonicalizationPatterns(
   });
 }
 
+//===----------------------------------------------------------------------===//
+// AtenCloneOp
+//===----------------------------------------------------------------------===//
+
+void AtenCloneOp::getCanonicalizationPatterns(RewritePatternSet &patterns,
+                                              MLIRContext *context) {
+  patterns.add(+[](AtenCloneOp op, PatternRewriter &rewriter) {
+    rewriter.replaceOp(op, op.getSelf());
```

Comment:
I think we need to make sure the type of `self` is a `torch.vtensor`. If this transformation is performed before all tensors have value semantics, it would not be correct.

Reply:
ok, done.